editors/vscode/syntaxes/lp.tmLanguage.json

```diff
@@ -70,13 +70,13 @@
     },
 
     "misc": {
-      "match": "\\b(assert|assernot|compute|print|proofterm|type)\\b",
+      "match": "\\b(assert|assertnot|compute|print|proofterm|type)\\b",
       "name": "keyword.control.query.lp"
     },
 
     "keywords": {
-      "match": "\\b(as|associative|begin|builtin|commutative|constant|debug|end|flag|in|infix|injective|left|off|notation|on|opaque|prover|prover_timeout|prefix|private|protected|quantifier|right|sequential|TYPE|unif_rule)\\b",
+      "match": "\\b(as|associative|begin|builtin|commutative|constant|debug|end|flag|in|infix|injective|left|off|notation|on|opaque|postfix|prover|prover_timeout|prefix|private|protected|quantifier|right|sequential|TYPE|unif_rule)\\b",
```

Comment:
Hi @gabrielhdt . The list is not ordered alphabetically (prefix, private, etc. should be before prover). Could you please fix it?

Reply:
I did it.